Codeinwp/neve
# fix: hide mobile sidebar toggle if sidebar is dropdown

### Summary
Hides mobile sidebar toggle if sidebar is dropdown. 


### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions


- Set the mobile sidebar to the dropdown mode.
- It should have only one toggle. None inside the actual sidebar.


Closes #723.
